src/app/interfaces/storage.interface.ts

```diff
@@ -21,6 +21,7 @@ export interface VDev {
   stats: TopologyItemStats;
   status: TopologyItemStatus;
   unavail_disk: unknown;
+  disk?: string;
```

Comment:
Does this field come from middleware? If we are adding this field on frontend, it's better to extend original interface and add new field there.

Reply:
This comes from middleware. For stripe vdevs, each vdev is only assigned one disk even if you add multiple disks in one stripe vdev, a new vdev is created for each disk. And that vdev has an empty `children` array and has `disk` as disk name.